tests/build_local_test_elfs.sh

```diff
@@ -12,32 +12,34 @@ mkdir -p elfs
 cd ..
 
 echo "*Building elfs for Nano S..."
+export BOLOS_SDK="$NANOS_SDK"
 
 echo "**Building app-paraswap for Nano S..."
-make clean BOLOS_SDK=$NANOS_SDK
-make -j DEBUG=1 BOLOS_SDK=$NANOS_SDK
-cp bin/app.elf "tests/elfs/paraswap_nanos.elf"
+make clean
+make -j DEBUG=1
+cp bin/app.elf "tests/elfs/plugin_nanos.elf"
 
 echo "**Building app-ethereum for Nano S..."
-cd $APP_ETHEREUM
-make clean BOLOS_SDK=$NANOS_SDK
-make -j DEBUG=1 BOLOS_SDK=$NANOS_SDK CHAIN=ethereum BYPASS_SIGNATURES=1 ALLOW_DATA=1
-cd -
+cd $APP_ETHEREUM || exit
```

Comment:
You can also define a `set -e` on top of the script if you want to exit on non zero commands.
I guess this script needs to stop as soon as one line fails, would it be `cd` or `make`.